kafka/channel/pkg/apis/messaging/v1alpha1/kafka_channel_defaults.go

```diff
@@ -34,7 +34,7 @@ func (c *KafkaChannel) SetDefaults(ctx context.Context) {
 		c.Annotations = make(map[string]string)
 	}
 	if _, ok := c.Annotations[messaging.SubscribableDuckVersionAnnotation]; !ok {
-		c.Annotations[messaging.SubscribableDuckVersionAnnotation] = "v1alpha1"
+		c.Annotations[messaging.SubscribableDuckVersionAnnotation] = "v1beta1"
```

Comment:
I am not so sure about this change here.
v1alpha1 Channelable duck type support is removed from eventing.

I think we should get rid of the the v1alpha1 channel resource completely, providing a migration script.

cc @vaikas for confirmation.

Reply:
Yes, I'd like us to get rid of the v1alpha1 shapes asap.